### ISSUE TITLE
Can't find static files

### ISSUE THREAD

**Issue by symonsoft** [marcgibbons/django-rest-swagger#772]
Python: 3.6
Django: 2.1
django-rest-swagger: 2.2.0
djangorestframework: 3.8.2

---

I have declared url for UI:

    schema_view = get_swagger_view(title='Swagger API')
    
    urlpatterns = [
        url(r'^docs/', schema_view),

And settings:

    INSTALLED_APPS = [
        'rest_framework',
        'rest_framework_swagger',
        ...
     ]

    SWAGGER_SETTINGS = {
        'USE_SESSION_AUTH': False,
    }

But it seems that django-rest-swagger static files are inaccessible:

Django log:

> [27/Aug/2018 13:56:10] "GET /docs/ HTTP/1.1" 200 3209
[2018-08-27 13:56:10] [26] [WARNING] [/usr/local/lib/python3.6/site-packages/django/utils/log.py:228] [log_response] Not Found: /static/rest_framework_swagger/bundles/vendors.bundle.css
[2018-08-27 13:56:10] [26] [WARNING] [/usr/local/lib/python3.6/site-packages/django/utils/log.py:228] [log_response] Not Found: /static/rest_framework_swagger/bundles/app.bundle.css
[27/Aug/2018 13:56:10] "GET /static/rest_framework_swagger/bundles/vendors.bundle.css HTTP/1.1" 404 130
[27/Aug/2018 13:56:10] "GET /static/rest_framework_swagger/bundles/app.bundle.css HTTP/1.1" 404 126
[2018-08-27 13:56:10] [26] [WARNING] [/usr/local/lib/python3.6/site-packages/django/utils/log.py:228] [log_response] Not Found: /static/rest_framework_swagger/logo_small.png
[2018-08-27 13:56:10] [26] [WARNING] [/usr/local/lib/python3.6/site-packages/django/utils/log.py:228] [log_response] Not Found: /static/rest_framework_swagger/bundles/vendors.bundle.js
[27/Aug/2018 13:56:10] "GET /static/rest_framework_swagger/logo_small.png HTTP/1.1" 404 118
[2018-08-27 13:56:10] [26] [WARNING] [/usr/local/lib/python3.6/site-packages/django/utils/log.py:228] [log_response] Not Found: /static/rest_framework_swagger/bundles/app.bundle.js
[27/Aug/2018 13:56:10] "GET /static/rest_framework_swagger/bundles/vendors.bundle.js HTTP/1.1" 404 129
[27/Aug/2018 13:56:10] "GET /static/rest_framework_swagger/bundles/app.bundle.js HTTP/1.1" 404 125

What I see:

![image](https://user-images.githubusercontent.com/12235978/44663878-22440980-aa2b-11e8-9d77-c7b1af1ae364.png)

**Comment by symonsoft** [comment]
I figured out that that happens if settings.DEBUG = False, but it's weird and should be fixed.

**Comment by wanglongze** [comment]
Did you solve this? I meet the same issue

**Comment by samsun387** [comment]
For anyone that run into this issue, you just need to run python manage.py collectstatic so that the bundle.css etc are generated for django to read.

**Comment by vsoch** [comment]
A third (very silly) alternative that I just ran into - I was updating an application that was using a previous version of django-rest-swagger, and I had copied the template into my own template folders to customize. At some point their static files changed to include a much smaller set (bundled I think) so I got the same (mostly empty page) not because I didn't copy the static, but because I was trying to import the wrong filenames (css and js) to begin with! Updating my custom template resolved the issue.

And the new styling is gorgeous! Thanks guys :)
